### PR TITLE
Drop Python2 support / maintenance reference GSPOE-36

### LINKED ISSUE #889: Drop python 2.7 support
Python 2.7 [stops receiving bug fixes after January 1, 2020](https://www.python.org/dev/peps/pep-0373/). By then we should drop python 2.7 support and all users should have migrated to python3 by then.

### PR DESCRIPTION
**Pull request by jwkaltz**
Fixes #889

### CODE REVIEW

**Comment by pfirpfel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ pyramid_oereb/lib/url.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/pyramid_oereb/pyramid_oereb/pullRequest?prid=5800622&bid=18990105)

File: pyramid_oereb/standard/load_legend_entries.py [hunk 1]
```diff
@@ -11,11 +10,7 @@
 
 from pyramid_oereb.lib.config import Config
 
-if sys.version_info.major == 2:
-    from urlparse import urlunsplit
-else:
-
-    from urllib.parse import urlunsplit
+from urllib.parse import urlunsplit
```
Comment (ger-benjamin) [inline pyramid_oereb/standard/load_legend_entries.py:13]: Can you group a little bit import (too much unnecessary blank lines) ?

File: tests/sources/test_address_geoadmin.py [hunk 2]
```diff
@@ -10,10 +9,7 @@
 from pyramid_oereb.contrib.sources.address import AddressGeoAdminSource
 from tests.mockrequest import MockParameter
 
-if sys.version_info.major == 2:
-    from urllib import urlencode
-else:
-    from urllib.parse import urlencode
+from urllib.parse import urlencode
```
Comment (ger-benjamin) [inline tests/sources/test_address_geoadmin.py:12]: Can you group a little bit import (too much unnecessary blank lines) ?

File: pyramid_oereb/contrib/print_proxy/mapfish_print.py [hunk 3]
```diff
@@ -23,10 +23,7 @@
 
 from .toc_pages import TocPages
 
-if sys.version_info.major == 2:
-    import urlparse
-else:
-    from urllib import parse as urlparse
+from urllib import parse as urlparse
```
Comment (ger-benjamin) [inline pyramid_oereb/contrib/print_proxy/mapfish_print.py:26]: Can you group a little bit import (too much unnecessary blank lines) ?

**Pull request merged**